changelog.md

```diff
@@ -147,34 +146,35 @@ class: changelog
 * Fixed the alignment of the icon in alert banner dialog components.
 * Fixed the visual blocks and visual char menu options not displaying their toggled state.
 * Fixed the editor not displaying as fullscreen when toggled.
+* Removed the tox-custom-editor class that was added to the wrapping element of codemirror.
 
 ## Version 5.0.0-preview-3 October 18, 2018
 * Changed editor layout to use modern CSS properties over manually calculating dimensions.
 * Changed `autoresize_min_height` and `autoresize_max_height` configurations to `min_height` and `max_height`.
 * Changed `Whole word` label in Search and Replace dialog to `Find whole words only`.
-* Fixed bugs with editor width jumping when resizing and the iframe not resizing to smaller than 150px in height.
+* Fixed bugs with editor width jumping when resizing and the iframe not resizing to smaller than `150px` in height.
 * Fixed mobile theme bug that prevented the editor from loading.
-* Fixed long toolbar groups extending outside of the editor instead of wrapping
+* Fixed long toolbar groups extending outside of the editor instead of wrapping.
 * Fixed dialog titles so they are now the proper case.
-* Fixed color picker default to be #000000 instead of #ff00ff.
-* Fixed "match case" option on the Find and Replace dialog is no longer selected by default.
-* Fixed vertical alignment of toolbar icons #DES-134
-* Fixed toolbar icons not appearing on IE11 #DES-133
+* Fixed color picker default to be `#000000` instead of `#ff00ff`.
+* Fixed "match case" option on the **Find and Replace** dialog is no longer selected by default.
+* Fixed vertical alignment of toolbar icons `#DES-134`.
```

Comment:
Not related to this change, but `#DES-134` and `#DES-133` below shouldn't be in here (they are JIRA numbers). Not sure how that was missed :disappointed:

Reply:
Removed JIRA numbers